### DISCUSSION TITLE
Code with syntax highlighting has low contrast

### DISCUSSION THREAD

**Discussion by leifg** [jpanther/congo#42] [General]
I am currently moving [my blog](blog.leif.io) over to a Hugo installation with Congo. I have various code snippets in different languages. But as soon as I use any syntax highlighting (adding the language to ```) the contrast in light mode makes it almost unreadable:

```
\```ruby
Conversation.where(status: "active")
\```
```

Will render like this in light mode:

<img width="656" alt="Screen Shot 2021-11-30 at 13 49 39" src="https://user-images.githubusercontent.com/110321/144133579-ae16e698-0a0f-4d0d-b6b9-9bf8513696bc.png">

And like this in dark mode:

<img width="655" alt="Screen Shot 2021-11-30 at 13 49 47" src="https://user-images.githubusercontent.com/110321/144133712-66fbb67f-8fed-4f19-a45d-9f34d9ade895.png">

Dark mode rendering is fine but notice how the light mode makes the `where` keyboard almost unreadable. 

Do I need to properly configure [Syntax Highlighting](https://gohugo.io/content-management/syntax-highlighting) for it to work in both modes?

Note: leaving out the language works fine, but I prefer having code with custom syntax highlighting.

**Comment by jpanther** [comment]
This looks like it's not using the correct configuration as the theme should be customising all these colours. I had a quick look at your repo and it seems like you're missing the `markup.toml` config file.

Specifically, you need to ensure you have the parameter below set to false otherwise it will default to using the Hugo standard syntax highlighting.

```toml
[highlight]
  noClasses = false
```

**Reply by leifg** [reply]
That worked like a charm. Thank you!
